src/docs/product/integrations/index.mdx

```diff
@@ -8,6 +8,10 @@ redirect_from:
   - /workflow/integrations/global-integrations/
   - /workflow/legacy-integrations/
   - /workflow/global-integrations/
+  - /integrations/azure-devops/?utm_source=blog&utm_m&code=swift
+  - /product/integrations/notification-incidents/discord/
+  - /sdks/react/integrations/plugin/
```

Comment:
It would be better to redirect this one here: https://docs.sentry.io/platforms/react-native/?utm_m=null&utm_source=blog